Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,20 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+//This can be removed when we move to VS2015
+#ifdef _MSC_VER
+	#define ThreadLocalStorage __declspec(thread)
+#elif !defined __ANDROID__ //Everything besides VS
+	#define ThreadLocalStorage __thread
+#elif defined __ANDROID__ //Android
+	#include <pthread.h>
+	static pthread_key_t s_tls_is_cpu_key = 0;
+#endif
```

Comment:
This code (and a couple of other bits) would be more clear if it you used #else; using if/elif without an else clause makes it nonobvious that every platform will use one of the codepaths.